noxfile.py

```diff
@@ -19,7 +19,24 @@
 
 
 # see https://pdm.fming.dev/latest/usage/advanced/#use-nox-as-the-runner
-os.environ.update({"PDM_IGNORE_SAVED_PYTHON": "1"})
+os.environ.update(
+    {
+        "PDM_IGNORE_SAVED_PYTHON": "1",
+    },
+)
+
+
+# load specific variables from the dotenv file if it exists
+def load_env_var(var: str, *, default: str, override: bool = False):
+    try:
+        import dotenv
+    except ImportError:
+        return default
```

Comment:
`os.environ` won't be set in this case, and the return value isn't being used anywhere.

Would it be easier to just do
```py
os.environ.setdefault(var, default)
```
at the top level and not do anything else? If someone wants to use their `.env`, `dotenv run` exists.

Reply:
https://github.com/DisnakeDev/disnake/pull/969/commits/7986a840205b6fb7186b03a02cafd0aa0823a801